### PR TITLE
Create recommendations from Paul (pmorto) for San Ramon

### LINKED ISSUE #7: Create a new file for pmorto's home town (San Ramon, CA)
# San Ramon, CA
- [x] Create a branch
- [x]  Make changes (commits)
- [x]  Open a Pull Request
- [x] Collaboration
- [x] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by pmorto**
closes #7

### CODE REVIEW

File: places/pmorto-sanramon.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# San Ramon, CA
+
+### Places to Eat
+- Thai House
+- Four Flavors
+- The Brass Door
```
Comment (chuoc) [inline places/pmorto-sanramon.md:6]: This place sounds interesting, any favorite :cocktail: ?

Comment (pmorto) [reply places/pmorto-sanramon.md:6]: Thai House is the best Thai restaurant in San Ramon and Danville. Four Flavors is interesting... it's like a Chipotle for Asian Fusion food. Really Good.

File: places/pmorto-sanramon.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### Places to Avoid
+- Most Mexican restaurants
+
+### Places to Go
+- Car Museum
```
Comment (sfrik) [inline places/pmorto-sanramon.md:12]: Is it american cars?

**Review by chuoc — APPROVED** [review]

**Comment by chuoc** [comment]
:+1:

**Pull request merged**